src/api/app/models/bs_request.rb

```diff
@@ -32,8 +32,10 @@ class SaveError < APIException
   scope :to_accept, -> { where(state: 'new').where('accept_at < ?', DateTime.now) }
   # Scopes for collections
   scope :with_actions, -> { joins(:bs_request_actions).distinct.order(priority: :asc, id: :desc) }
+  scope :with_reviews, -> { joins(:reviews) }
```

Comment:
Nitpick, but I am not sure if this justifies a scope. It's not like we are wrapping a complex operation here.